netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -129,6 +134,16 @@ public NettyChannelBuilder flowControlWindow(int flowControlWindow) {
     return this;
   }
 
+  /**
+   * Sets the maximum message size allowed to be received on the channel. If not called,
+   * defaults to 100 MiB.
```

Comment:
This class already has several imports referencing the internal package, why not just reference it in the javadoc?